web/src/styles/global.css

```
@@ -0,0 +1,739 @@
+  --quiet: rgba(var(--surface-rgb), 0.55);
+  --faint: rgba(var(--surface-rgb), 0.38);
+  --line: rgba(var(--surface-rgb), 0.1);
+  --line-strong: rgba(var(--surface-rgb), 0.2);
+  --card-bg: rgba(var(--surface-rgb), 0.025);
+  --card-bg-hover: rgba(var(--surface-rgb), 0.04);
+  --tile-bg: rgba(var(--surface-rgb), 0.03);
+  --nav-bg: rgba(0, 0, 0, 0.9);
+
+  font-family: 'Geist Mono Variable', ui-monospace, 'SF Mono', 'Fira Code', Menlo, Consolas, monospace;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.12.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)